roles/self_hosted_additional_host/tasks/main.yml

```diff
@@ -11,16 +11,6 @@
   tags:
     - packages
 
-- name: get the provisioning nic for the machine
-  shell: |
-     ip addr | grep -Pzo '^[0-9]+:(.*):.*\n.*{{ mac_address }}.*' | sed -r -n 's/[0-9]+: (.*):.*/\1/p' | head -n1
```

Comment:
We added this when we hit issues on bare metal with multiple nics, where we only knew the mac address of the correct nic. I'm not totally sure how `ansible_default_ipv4.interface` works, but could there be issues with multiple nics?